tools/ci_build/github/azure-pipelines/templates/linux-cpu-packaging-pipeline.yml

```diff
@@ -4,8 +4,26 @@ parameters:
   type: string
   default: ''
 
+- name: stage_name_suffix
+  displayName: job name for nuget
```

Comment:
I don't quite understand this description. looks like this value is part of the stage name.